### PR TITLE
QUICK-120: Give DBs more time to handle the XL dataset

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt [hunk 1]
```diff
@@ -34,7 +34,7 @@ private val JIRA_XL_DATASET = StorageLocation(
                 artifactName = "database.tar.bz2",
                 location = location,
                 unpackedPath = "database",
-                downloadTimeout = Duration.ofMinutes(40)
```
Comment (wyrzyk) [inline src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt:37]: How large is the zip?

Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt:37]: 24 GB

Comment (wyrzyk) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt:37]: You can use `HttpDatasetPackage` to download and unpack the resource faster (you would also have to use an ephemeral drive for the database). See:
https://github.com/atlassian/infrastructure/commit/57925df625ff0d7b1cb93fdcf011141bab694a3a

Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt:37]: That's on our radar. However, HTTP transport requires public read access - correct?
If so, we gotta sanitise the dataset first.

Comment (wyrzyk) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt:37]: > If so, we gotta sanitise the dataset first.

There's also a solution that can fit in a middle. You can use pre-signed URLs. The URLs will be hard to guess and available only temporarily. 

https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html

**Pull request merged**